### ISSUE TITLE
NextCloud backup script fix

### ISSUE THREAD

**Issue by scottyeager** [threefoldtech/info_grid#636]
The NextCloud guide has instructions to back up to a second VM. They include a script to run on the second VM:

```
#!/bin/bash

sudo rsync -avz --progress --delete --log-file=/root/nextcloud_backup/rsync_nextcloud_storage.log /root/nextcloud_backup/ root@<Nextcloud_VM_IP_Address>:/mnt/backup
```

We got a report that this has the source and destination switched, such that backups on the original VM will actually be deleted. Seems it should be:

```
#!/bin/bash

sudo rsync -avz --progress --delete --log-file=/root/nextcloud_backup/rsync_nextcloud_storage.log root@<Nextcloud_VM_IP_Address>:/mnt/backup /root/nextcloud_backup/ 
```

**Comment by Mik-TF** [comment]
Fixed in #635

**Comment by Mik-TF** [comment]
Good catch thanks.
